hack/local-up-cluster.sh

```diff
@@ -74,6 +74,7 @@ ENABLE_DAEMON=${ENABLE_DAEMON:-false}
 HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"127.0.0.1"}
 EXTERNAL_CLOUD_PROVIDER=${EXTERNAL_CLOUD_PROVIDER:-false}
 EXTERNAL_CLOUD_PROVIDER_BINARY=${EXTERNAL_CLOUD_PROVIDER_BINARY:-""}
+EXTERNAL_CLOUD_VOLUME_PLUGIN=${EXTERNAL_CLOUD_VOLUME_PLUGIN:-""}
```

Comment:
Can you please default `EXTERNAL_CLOUD_VOLUME_PLUGIN` to `CLOUD_PROVIDER` when `EXTERNAL_CLOUD_VOLUME_PLUGIN` is not set?

Reply:
That would be great!